packages/browser/src/plugins/remote-loader/__tests__/action-destination.test.ts

```diff
@@ -0,0 +1,111 @@
+import unfetch from 'unfetch'
+import braze from '@segment/analytics-browser-actions-braze'
+
+import { AnalyticsBrowser } from '../../../browser'
+import { createSuccess } from '../../../test-helpers/factories'
+import { JSDOM } from 'jsdom'
+
+jest.mock('unfetch')
+jest.mocked(unfetch).mockImplementation(() =>
+  createSuccess({
+    integrations: {},
```

Comment:
Nit: Seems like a good place to use the CDNSettingsBuilder helper class!

Reply:
updated